content/en/docs/reference/rules/supported-events/index.md

```diff
@@ -11,7 +11,7 @@ Here are the system call event types and args supported by the [kernel module an
 Note that several event types exist:
 * [Syscall events](#syscall-events) correspond to Linux system calls. Most of them have parameters, documented below, while some are detected as generic and they only offer the syscall ID.
 * [Tracepoint events](#tracepoint-events) represent internal kernel events that may be significant but don't directly translate to any syscall.
-* [Metaevents](#metaevents) are internal Falco events. They are generally not available for rules.
+* [Metaevents](#metaevents) are internal Falco events.
```

Comment:
```suggestion
* [Metaevents](#metaevents) are generated from supplementary data sources, for instance, during data enrichment procedures or when the need for asynchronous actions arises. This group also encompasses some of Falco's internally produced events (such as the `drop` event) that are unavailable for rules.
```

IIRC, only `EC_INTERNAL` are filtered out. Any other metaevent goes thru the rule engine as common events.